docs/sources/operations/storage/tsdb.md

```diff
@@ -0,0 +1,75 @@
+---
+title: TSDB
+---
+# TSDB(Time Series Database) Index Store
+
+:warning: TSDB is still an experimental feature. It is not recommended to be used in production environments.
+
+With TSDB index store, you can run Grafana Loki without any dependency on NoSQL stores for storing index.
+Using the TSDB index store, you can run Grafana Loki without any dependency on NoSQL stores for storing indexes. It works the same way as the [boltdb-shipper](./boltdb-shipper) index store, but with reduced total cost of ownership, and better query performance.
+Loki uses [Prometheus TSDB format](https://github.com/prometheus/prometheus/tree/main/tsdb#tsdb) for storing the index with 
+some tweaks on top of it for query planning.
+
+## Example Configuration
+
+Example configuration with GCS:
+
+```yaml
+schema_config:
+  configs:
+    - from: 2018-04-15
+      store: tsdb
+      object_store: gcs
+      schema: v12
+      index:
+        prefix: loki_index_
+        period: 24h
+
+storage_config:
+  gcs:
+    bucket_name: GCS_BUCKET_NAME
+
+  tsdb_shipper:
+    active_index_directory: /loki/index
+    shared_store: gcs
+    cache_location: /loki/tsdb-cache
+```
+
+This would run Loki with TSDB index storing TSDB files locally at `/loki/index` and chunks at configured `GCS_BUCKET_NAME`.
+It would also keep shipping TSDB files periodically to the same configured bucket.
+It would also keep downloading TSDB files from shared bucket uploaded by other ingesters to `/loki/tsdb-cache` folder locally.
```

Comment:
```suggestion
It would also keep downloading TSDB files from the shared bucket uploaded by other ingesters to the local `/loki/tsdb-cache` folder.
```